taxonomy/models.py

```diff
@@ -610,3 +611,16 @@ class Meta:
         app_label = 'taxonomy'
         verbose_name = 'Skill Subcategory'
         verbose_name_plural = 'Skill Subcategories'
+
+
+class SkillsQuiz(TimeStampedModel):
```

Comment:
We can not rely on discovery to have all the user records, as that information is stored in LMS so we are using username here instead of a foreign key. And the reason it is not a unique field is that a user is allowed to fill out and save multiple entries of the skills quiz, we can find the latest one based on `created_at`.